tests/alfacase/test_generate_case_schema/test_generate_schema_for_all_cases.txt

```diff
@@ -338,6 +351,14 @@ open_hole_description_schema = Map(
     }
 )
 
+overall_pipe_trend_description_schema = Map(
+    {
+        "curve_names": Seq(Str()),
+        "location": Enum(['main', 'annulus', 'not_defined']),
+        Optional("element_name"): Str(),
```

Comment:
If you follow the comments about this no longer being optional, make sure to update this.

Reply:
Sure, the schema is generated by `inv cog` and a `--force-regen` do the work :)